lib/devices/android/chrome.js

```diff
@@ -121,11 +117,15 @@ ChromeAndroid.prototype.pushAndUnlock = function (cb) {
 };
 
 ChromeAndroid.prototype.createSession = function (cb) {
+  cb = _.once(cb);
   var caps = {
     chromeOptions: {
       androidPackage: this.args.appPackage
     }
   };
+  if (this.args.enablePerformanceLogging) {
+    caps.loggingPrefs = {performance: 'ALL'};
+  }
 
   var knownPackages = ["org.chromium.chrome.shell",
```

Comment:
Out of scope for this ticket, but we should consolidate these collections with those in appium.js and put it in appium-support or some such.